src/module-analyzer.js

```diff
@@ -274,6 +275,9 @@ export class ModuleAnalyzer {
         } else if (conventional = BindingBehaviorResource.convention(key)) {
           resources.push(new ResourceDescription(key, exportedValue, conventional));
           metadata.define(metadata.resource, conventional, exportedValue);
+        } else if (conventional = ViewEngineHooksResource.convention(key)) {
```

Comment:
@davismj In fact, I think you can do a little refactor here starting on https://github.com/davismj/templating/blob/6cf33d84b7ded2d5b3cefcb9bc79b3f02c42e3d8/src/module-analyzer.js#L272. 

```
else if (conventional = 
  ValueConverterResource.convention(key) 
  || BindingBehaviorResource.convention(key) 
  || ViewEngineHooksResource.convention(key)
) {
```
this looks like a nice refactor, don't you agree?